certified-connectors/IN-D Payables/apiProperties.json

```diff
@@ -1,4 +1,4 @@
-{
+  {
```

Comment:
@gowthamd3 The `iconBrandColor` seems different in the original file you have submitted to us in the cert. portal. That is "#048c88".

Reply:
@sriyen-msft  done the change you mentioned.